### PR TITLE
GerardRed_cbds_antenna_gain

### PR DESCRIPTION
**Pull request by redtechnologies**

### CODE REVIEW

File: src/prop_current/cbsd_antenna_gain.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+'''
```
Comment (sushantparkhi) [inline src/prop_current/cbsd_antenna_gain.py]: can you add this doctring at the top of the file
https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/blob/master/src/harness/sas.py#L1

File: src/prop_current/cbsd_antenna_gain.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Created on 6 juil. 2017
+
+@author: Red Technologies
+Gerard Garnier
+'''
+
+def GetCBSDAntennaGain(AZ, antennaAzimut, antennaBeamWidth):
```
Comment (sushantparkhi) [inline src/prop_current/cbsd_antenna_gain.py]: May be the use of Camel Case notation?

File: src/prop_current/cbsd_antenna_gain.py (hunk 1 above)
Comment (sushantparkhi) [inline src/prop_current/cbsd_antenna_gain.py]: Can you move this file to harness/reference_models/{$folder_name} https://docs.google.com/document/d/17ZvP0Ju8eENJ4jRKx0c2WL7yCwX22J8xjaGA93Cc_FI/edit#heading=h.ul0kthrnnhpb

File: src/prop_current/cbsd_antenna_gain.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    """
+    Returns the CBSD antenna gain (R2-SGN-20).
+    Inputs:
+        AZ                        azimuth angle (deg : 0 - 359)
+        antennaAzimut             (deg : 0 - 359)
```
Comment (sushantparkhi) [inline src/prop_current/cbsd_antenna_gain.py]: antennaAzimuth?

**Comment by sbdt-google** [comment]
Marking as deprecated as the code has been extended and integrated with reference propagation models in PR: https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/pull/283

**Pull request closed without merging**